src/transformation/Transformation.ts

```diff
@@ -231,10 +230,11 @@ class Transformation {
   }
 
   /**
-   * @param {SmartObject} smartObject
+   * @description Adds a layer in a Photoshop document.
+   * @param action
    */
-  getSmartObject(smartObject: SmartObject): this {
-    return this.addAction(smartObject);
+  psdTools(action: IAction): this {
```

Comment:
Can this accept the right type?

it can only accept one of the PSD actions, right?